docs/guides/cli.md

```diff
@@ -1 +1,174 @@
+
+-   **--target** <em>(str, default: localhost with chat completions api for VLLM)</em>: Target for benchmarking 
+    
+	-   optional breakdown args if target isn't specified:
+    
+		-   **--host** <em>(str)</em>: Host URL for benchmarking
+    
+		-   **--port** <em>(str)</em>: Port available for benchmarking
+    
+-   **--backend** <em>(str, default: server_openai [vllm, TGI, llama.cpp, DeepSparse, and many popular servers match this format])</em>: Backend type for benchmarking
+    
+-   **--model** <em>(str, default: auto populated from vllm server)</em>: Model being used for benchmarking, running on the inference server
+        
+-   **--task** <em>(str), optional)</em>: Task to use for benchmarking
+
+-  **--output-path** <em>(str), optional)</em>: Path to save report report to
+
+
+
+#### Workload Data
+
+This section of input parameters covers the data arguments that need to be supplied such as a reference to the dataset and tokenizer. The list of arguments and their defintions are presented below:
+
+-   **--data** <em>(str)</em>: Data file or alias for benchmarking
+
+-   **--data-type** <em>(ENUM, default: emulated; [file, transformers])</em>: The type of data given for benchmarking
+
+-   **--tokenizer** <em>(str)</em>: Tokenizer to use for benchmarking
+
+#### Workload Type
+
+This section of input parameters covers the type of workload that you want to run to represent the type of load you expect on your server in production such as rate-per-second and the frequency of requests. The full list of arguments and their definitions are presented below:
+
+-   **--rate-type**  <em>(ENUM, default: sweep; [serial, constant, poisson] where sweep will cover a range of constant request rates and ensure saturation of server, serial will send one request at a time, constant will send a constant request rate, poisson will send a request rate sampled from a poisson distribution at a given mean) </em>: Type of rate generation for benchmarking
+    
+-   **--rate** <em>(float)</em>: Rate to use for constant and poisson rate types
+    
+-   **--max-seconds** <em>(integer)</em>: Number of seconds to result each request rate at
+    
+-   **--max-requests** <em>(integer)</em>: Number of requests to send for each rate
+    
+### Output Metrics via GuideLLM Benchmarks Report
+
+Once your GuideLLM run is complete, the output metrics are displayed as a GuideLLM Benchmarks Report via the Terminal in the following 4 sections: 
+
+- **Requests Data by Benchmark**
+- **Tokens Data by Benchmark**
+- **Performance Stats by Benchmark**
+- **Performance Summary by Benchmark**
+
+The GuideLLM Benchmarks Report surfaces key LLM metrics to help you determine the health and performance of your inference server. You can use the numbers generated by the GuideLLM Benchmarks Report to make decisions around server request processing, Service Level Objective (SLO) success/failure for your task, general model performance, and hardware impact.
```

Comment:
Does guidellm provide healthchecking or health validation? Is this more of a roadmap goal? Or do you plan to inspect vllm and other backend directly to determine helath.

"to make decisions" may be better worded as "to evaluate".
You can use the numbers generated -> use the benchmark metrics"